.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             pip install .
             pip install --pre jaxlib \
               -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html
-            pip install libtpu-nightly \
+            pip install --pre libtpu-nightly \
```

Comment:
optional: Maybe also add `-U`?

Reply:
Done. It's not necessary since we uninstall at the beginning, but shouldn't hurt. Also added to jaxlib for consistency.